src/utils.rs

```diff
@@ -542,6 +551,37 @@ pub fn sort_with_jaccard_as_tie_breaker(
     vec_to_sort
 }
 
+// TODO: Test for this function
+pub fn import_custom_ic_map(path: &PathBuf) -> Result<HashMap<String, f64>, io::Error> {
+    let file = fs::File::open(path)?;
+    let reader = BufReader::new(file);
+    let mut ic_map: HashMap<String, f64> = HashMap::new();
+
+    for line_result in reader.lines() {
+        let line = line_result?;
+        let columns: Vec<&str> = line.split('\t').collect();
+
+        if columns.len() >= 2 {
```

Comment:
Yes right now we are just parsing the first 2 columns and ignoring the others. We could add functionality for the user to provide the columns of interest but since the file does not have column names, it may need index provision which could get confusing?